cpp/tests/transform/integration/unary_transform_test.cpp

```diff
@@ -335,4 +336,51 @@ __device__ inline void transform(
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*ptx_result, expected);
 }
 
+template <typename T>
+struct TenaryDecimalOperationTest : public cudf::test::BaseFixture {};
```

Comment:
Is this a typo? Is it supposed to be “Ternary”? However, I don’t see a ternary operator (?) here. Maybe `DecimalArithmeticOperationTest` is a better name.

Reply:
The intention here is to test that both multi-input column and decimals work together.